git_repo/services/ext/gerrit.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+from ..service import register_target, RepositoryService
+
+@register_target('gerrit', 'gerrit')
+class GerritService(RepositoryService):
+    fqdn = "localhost"
+    port = 29418
+    _max_nested_namespaces = None
+
+    def connect(self):
+        pass
+
+    @classmethod
+    def get_auth_token(cls, login, password, prompt=None):
+        return password
```

Comment:
doesn't gerrit support private token keys ? It'd be cool to avoid storing passwords in config files, but revokable tokens are 👌

Reply:
For rest API gerrit uses HTTP basic authentication with the HTTP password from the user’s account settings page. HTTP password is also used for cloning using http. So http password is something like private token here, not an actual account password. By the way, it can't be set by user. It is generated on separate settings page. 
More info here: https://gerrit-review.googlesource.com/Documentation/rest-api.html#authentication

I expected that users will provide HTTP password each time they setup access to Gerrit.

Reply:
ok, as long as the secret is revocable and not likely to be used elsewhere by the user, it's all good for me 👍